src/styles/components/_component-card.scss

```diff
@@ -86,11 +86,18 @@
     }
   }
 
-  &:focus .component-card {
+  // tabbed focus has rotation
+  &:focus-visible .component-card {
     position: relative;
     z-index: 1;
     outline: 2px solid var(--accent-color);
     outline-offset: 2px;
     transform: rotate(2deg);
   }
+
+  // click focus has no rotation
+  &:focus:not(:focus-visible) .component-card {
+    outline: 2px solid var(--accent-color);
+    outline-offset: 2px;
+  }
 }
```

Comment:
> Keeping that tabbed focus, with your cursor, click on a different component. When you click, you should not see the component rotate.

Based on the validation instructions I did as instructed, when I focus with the keyboard on a component but then click on another component with the cursor it does rotate back.  Did you mean the original focused component shouldn't rotate back or the component that is clicked on should not rotate?

Is this the correct behavior you needed us to test?
![component focus click](http://g.recordit.co/uj2YtPtLlm.gif)

Reply:
Yes! Before the mouseclick was _also_ rotating the cards. We wanted to get rid of that. I think it's ok that the tabbed focus un-rotates.

Reply:
I think it is too, I just wanted to verify I understood which one should be moving or not moving on click. lol